test-service/deploy/ansible/README.adoc

```diff
@@ -51,3 +51,22 @@ image::images/kiali-test-depth-sink.png[]
 
 === Kiali Test Breadth Sink
 image::images/kiali-test-breadth-sink.png[]
+
+
+= Kiali Test Mesh Multi-Namespace Ansible Installer
+
+A straightforward instalation command is
+
+[source,shell]
+----
+ansible-playbook deploy_test_meshes.yml -e number_of_services=5 -e number_of_versions=2 -e number_max_of_namespaces=2 -e '{"meshes": ["kiali-test-depth", "kiali-test-breadth", "kiali-test-circle", "kiali-test-circle-callback", "kiali-test-hourglass", "kiali-test-depth-sink", "kiali-test-breadth-sink"]}' -v
+----
+which
+
+- `number_of_services` is the number of services will be deployed per namespace
+
+- `number_of_versions` is the number of versions each service will have per namespace
+
+- `meshes` specifies which mesh will be installed.
+
+- `number_max_of_namespaces` is the number max of namespaces. The playbook will randomize the choice of namespace for each service (eg: in 2 namespaces of max with 4 services, we could have scenario 1 with: 3 services at namespace-1 and 1 service at namespace-2 or we could have scenario 2 with 2 services on each namespace).
```

Comment:
I think this "random" aspect is not going to be useful. This should be more repeatable and more fixed. Think about why we are using these test meshes. We are building these test meshes to run tests and make sure things work correctly. But suppose we see a bug in Kiali that was exposed by this multi-namespace test mesh. We write a bug report and tell the developers about it. The developers are immediately going to ask "how can I replicate this bug?" You can tell them "run the multi-namespace test mesh" but if it is all random, the developer is not guaranteed to get the same test mesh that YOU got when you saw the bug. So it is possible the developer might not be able to replicate the bug since the test mesh isn't going to be guaranteed to be the same.

We should have this more repeatable - I think we should take out the randomness because otherwise won't it be hard to replicate test results?

Reply:
I think both have a case to exist. The random one is like a fuzzing, we can discover some edge cases using it, but the predictable one (that should be default) is to show the expected behaviour.

Maybe we should move the random one to a feature flag?